modules/ROOT/pages/changelog.adoc

```diff
@@ -5,6 +5,22 @@
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
 
+== 6.7.2 - 2023-10-25
+
+=== Fixed
+* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
+//TINY-10154
+* The function `getModifierState` did not work on events passed through the editor as expected.
+//TINY-10263
+* List items containing a nested list between two text caused some list operations to fail.
+//TINY-10268
+* css loading in `toolbardrawertoggletest` flickering.
+//TINY-10249
+* Toggling a list that contained a list item element — `<li>` — which, in turn, contained another list item element as its first child, removed other content within the first list item element.
```

Comment:
Hard to describe that weird scenario:
```suggestion
* Indenting or outdenting a list item that had non list item siblings after it would remove those siblings.
```

Reply:
```suggestion
* Indenting or outdenting a list item that contained a non list item siblings after it would result in those siblings being removed.
```

Reply:
@spocke Ive updated the copy edit suggestion, thoughts?

Reply:
```suggestion
* Indenting or outdenting a list item that contained non list item siblings after it would result in those siblings being removed.
```